client/js/lounge.js

```diff
@@ -1142,13 +1207,34 @@ $(function() {
 			var chan = $(this);
 			if (chan.find(".messages .msg:not(.unread-marker)").slice(0, -100).remove().length) {
 				chan.find(".show-more").addClass("show");
+
```

Comment:
Ah right, sorry I missed that the issue only happens because the setInterval exists. Alright, Ignore this then, that's a seperate PR.